extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js

```diff
@@ -0,0 +1,25 @@
+require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracker) {
+	'use strict';
+
+	var track = function (data) {
+		tracker.track(window.Object.assign({
+			action: tracker.ACTIONS.CLICK,
+			category: 'rwe-page-header',
+			trackingMethod: 'analytics'
+		}, data));
+	};
+
+	$(function () {
+		$('.rwe-page-header--dropdown').hover(function () {
+			$(this).addClass('rwe-page-header--active');
+		}, function () {
+			$(this).removeClass('rwe-page-header--active');
+		});
+
+		$('.rwe-page-header-nav__link').on('click', function (e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "e". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)